api/server/controllers/ModelController.js

```diff
@@ -6,6 +6,7 @@ const {
   ListAgentsCommand,
   ListAgentAliasesCommand,
 } = require('@aws-sdk/client-bedrock-agent');
+const User = require('~/models/User');
```

Comment:
I feel like using a tilde(~) here is possibly a bad idea, it's very hard to have the context of all the 400 changed files so I might not know the whole story but how can we be sure that we're always going to be storing models/ in the home directory? What exactly is models/User? 
In our docker builds we keep all the source in app/ so to me this is strange.

Why are we deviating from the standard way of importing here?